conf/test.config

```diff
@@ -45,7 +45,7 @@ params {
     // Test dataset is too small for the nsub default value
     deseq2_vst_nsub                   = 500
 
-    // Activate GSEA
-    gsea_run = true
-    gsea_gene_sets = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
+    // Activate gprofiler2
```

Comment:
Could you not activate this in the small test profile please? I left GSEA out of this to keep it minimal. Fine for test_full.

Also, when I tried to run GSEA at the same time by activating:

```
   // Activate GSEA
    gsea_run = true
    gene_sets_files = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/mus_musculus/gene_set_analysis/mh.all.v2022.1.Mm.symbols.gmt'
```

I got this error:

```
  Your custom annotations ID is gp__NSxz_Dr8C_16U
  You can use this ID as an 'organism' name in all the related enrichment tests against this custom source.
  Just use: gost(my_genes, organism = 'gp__NSxz_Dr8C_16U')
  Detected custom GMT source request
  No results to show
  Please make sure that the organism is correct or set significant = FALSE
```

Reply:
Hmm, sorry Jonathan, I can't reproduce that error...Did you just run the test.config with the `gsea_run` and `gene_sets_files` parameters added?

Reply:
Yep!

Reply:
I added both gsea and gpro to the full test (this time with a token for gpro from a previous run; let's see how long g:profiler saves these tokens). For me it worked; can you try again?